docs/proposals/20220927-label-sync-between-machine-and-nodes.md

```diff
@@ -0,0 +1,131 @@
+---
+
+title: Label Sync Between Machine and underlying Kubernetes Nodes
+authors:
```

Comment:
nit: might be good to add previous authors and/or reviewers
(not sure how this should be handled, feel free to resolve)